src/components/headers/formHeader/FormHeaderButton.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { GestureResponderEvent, Pressable, Text } from 'react-native'
+
+interface FormHeaderButtonProps {
+  onPress?: ((event: GestureResponderEvent) => void) | null | undefined
+  label?: string
+  iconLeft?: React.ReactNode
+  iconRight?: React.ReactNode
+  disabled?: boolean
+}
+
+function FormHeaderButton(props: FormHeaderButtonProps) {
+  return (
+    <Pressable
+      disabled={props.disabled}
+      onPress={props.onPress}
+      className="flex flex-row justify-center"
+    >
+      {props.iconLeft && <>{props.iconLeft}</>}
```

Comment:
You don't have to use a fragment here, can you just use this?
`{props.iconLeft}` and `{props.iconRight}`